.eslintrc

```diff
@@ -14,7 +14,8 @@
     "consistent-return": 0,
     "comma-dangle": 0,
     "spaced-comment": 0,
-    "global-require": 0
+    "global-require": 0,
+    "no-param-reassign": ["error", {"props": false}]
```

Comment:
This is not necessarily the right choice, the way I see it the places I saw it you have currently been hacking around an eslint rule without actually following what the purpose of it is, instead of just disabling it.

So either you should add this rule to eslint and not follow the airbnb style guide completely which I guess I'm personally okay with in the cases I have seen in the code or we should actually follow the rule.

To explain what I mean, what I have seen below which I'll now write about here instead of commenting on it in the actual examples, is that there is an object parameter to a function that is for example named `target`. The first line of the code will then be `const element = target` followed by a lot of reassignments of properties of element. But since all objects are pointers in Javascript you are still changing the props of the parameter, eslint just isn't picking up on it now because you tricked eslint. 

[This](https://github.com/airbnb/javascript#functions--mutate-params) is what airbnb had in mind, and I guess that makes some sense to me in the sense that this is super safe programming standards in general, but in our case when we're for example doing a `.forEach` to mutate an array of objects it doesn't have any logical sense to copy the whole object and then replace the old one, when we actually want to mutate properties. So in this case I guess I argue for the case of adding this rule to our linter?